src/views/Header/HeaderTitle.js

```diff
@@ -9,7 +9,7 @@ const HeaderTitle = ({ style, ...rest }) => (
     {...rest}
     style={[styles.title, style]}
     accessibilityTraits="header"
-    accessible={false}
+    accessible={true}
```

Comment:
BTW why do we need to specify this prop at all? I was under the impression that `Text` elements were handled automatically?

Reply:
Yeah. I just changed false to true and left if for better visibility

Reply:
I just wonder why someone changed this to false

Reply:
Seems author isn't on GH anymore: https://github.com/react-navigation/react-navigation-stack/commit/1931b5b9688007fa1316aef0023b6546a931e387

Reply:
@satya164 Do you think this can be merged now? It looks like @chrisburgs is still on GH, author of https://github.com/NewtopiaDev/react-navigation-stack/pull/1 is still around. Or maybe @Nigh7Sh4de can shed some light on it?

Reply:
I want to be sure if `accessible={true}` is necessary here. iirc it actually makes children inaccessible.

Reply:
`accessible={false}` is default afaik so not sure setting it explicitly is necessary

Reply:
I did some quick tests and setting this prop on the `Text` element doesn't seem to do anything. Can you remove this prop to avoid confusion in future and I'll merge? @MagTuxGit